Cargo.toml

```
@@ -1685,10 +1684,11 @@ wasm = false
 name = "system_stepping"
 path = "examples/ecs/system_stepping.rs"
 doc-scrape-examples = true
+required-features = ["bevy_debug_stepping"]
 
 [package.metadata.example.system_stepping]
 name = "System Stepping"
-description = "Demonstrate stepping through systems in order of execution"
+description = "Demonstrate stepping through systems in order of execution. Requires enabling the `bevy_debug_stepping` feature."
```

```suggestion
description = "Demonstrate stepping through systems in order of execution."
```
It's not needed to add it here, it's already mentioned in the required features of the example